### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by Azvalk** [Ladysnake/Requiem#423] [Ideas]
Hi ^^
I love your mod, the idea is perfect, and so I started to test it with some others..

So I have some suggestions for you =)
- Could it be possible to have a normal player speed while possessing a monster ?
Or a config for that maybe ?

- Would it be possible to add blocks in the config, to make these blocks traversable in soul form ?

- Would it be possible, when possessing a monster, to quit him volontary, with a keybind ?
( Config to allow this or not )

I continue to test with many other mods that I like ^^
there is many strange things that happen, which doesn't happen with only dissolution installed

**Comment by Pyrofab** [comment]
- The speed is not something we intend to change for now. Even in vanilla, possessing a baby zombie or a wither skeleton should alleviate this issue.
- For performance and simplicity reasons, I only check the thickness of collision boxes, not their content. As such, adding a config for which blocks are traversable is not planned for now. I could however add a config for that maximum wall thickness.
- Requiring the mob to die to end possession is currently part of the design. We may possibly add an upgrade allowing you to leave mobs in the future.

**Comment by Azvalk** [comment]
Other ideas, more advanced/complex, to enhance soul gameplay.

Evolution of our soul.
Maybe a skill tree or just distinct upgrades by some rituals.

It could make some things possible.
For exemple: 
- possessing other monster type than undead.
( The vanilla ones should suffise )
- upgrade the fly speed
- levitate an item on the ground, to play the ghost '-'
- push an entity ( monster, animal, player (depend of pvp rule on server) )
- make ghost noises

hm.. if your add 2 things it could became even more interesting..
- Weak souls
- Animal / Monster Souls

Imagine: you kill a cow, or a witch, or a creeper, you see nothing special, but the creature's soul is released in the air, invisible to you.
But, if your are in soul form, you can see it, attack it, and eat it ^^ 
And it can serv the progression of your soul.

Weak souls could be natural wandering souls already in the air, without the need of killing something, but it will not provide as much "soul energy" than a creature soul.

Thank for responding to previous suggestions ^^
The more important one was the possibility to end the possession, but I only propose this to allow the customisation of the difficulty.
When you find a cool mod, the first thing you try is to customize it ^^'
But, I saw that the eau de mort work when we possess something.

**Comment by ruan128** [comment]
a ideia if u take dropped iron was a iron golem u regen health also u can put in possesives "#requiem:living" to posses alive mobs and also players and also add a new soul type ghost to who dont know how to config and also fix the bug that when u remove ender dragon on black list u cant posses and the bug that u cant control wither powers and cant fly was wither also this mod is an of the best for me also when i put a enderman on the config to can posses and try to posses it i get tp to him and nothing else happen
